app/views/idv/shared/_ssn.html.erb

```diff
@@ -1,6 +1,6 @@
 <% title t('titles.doc_auth.ssn') %>
 
-<% if should_render_id_success_alert %>
+<% if id_success_alert_enabled %>
```

Comment:
QQ, what does the `id_` mean here? would it mean the same thing if we called it `success_alert_enabled`?

Reply:
It means that the success alert is about the user's ID being verified. Not necessary though, if we need to add another success alert later we can differentiate then

Reply:
Updated in da36d11064149805774a30684df69fbc3e1a06aa